src/common/playlists/PlayMeta.jsx

```diff
@@ -0,0 +1,53 @@
+import { cloneElement } from "react";
+import { Helmet } from "react-helmet";
+
+function PlayMeta({ id, name, description, path, cover, component }) {
+  const playFolder = path.split("/")[2];
+
+  let metaImage; // Initialize metaImage variable
+  if (cover) {
+    metaImage = cover; // If cover path is given, use that
+  } else {
+    try {
+      // If not, try finding the cover.png in the play's folder
+      metaImage = require(`../../plays/${playFolder}/cover.png`);
+    } catch {
+      // If no image is available, cover stays as undefined
+      console.log("No cover available.");
```

Comment:
If no image is available, let us set the top-level meta image than keep it blank? DefMeta will take care of it already?

Reply:
Ah, I knew I forgot something! I'll implement that ASAP, I think simply just not setting a meta tag if the metaImage variable is undefined should fix this!

Reply:
Done! Now, if a play doesn't have a cover image linked to it, it'll just display the default one